README.md

```diff
@@ -225,18 +195,49 @@ If you wish to use a github app,
    ```
    GITHUB_APP_ID=<app id>
    ```
- - Generate a private key.  A pem file will download.  You need to use the content of this file in your .env file:
+ - Generate a private key. A pem file will download. You need to use the content of this file in your .env file:
    ```
    GITHUB_APP_PRIVATE_KEY=-----BEGIN RSA PRIVATE KEY-----\nMIIEpQ......\n-----END RSA PRIVATE KEY-----
    ```
  - Install the app (follow instructions in link above) to your organization for "All repositories"   
    
 
 
+# Local S3 emulation with Minio
+
+Our `docker-compose` configuration includes an instance of [Minio](https://github.com/minio/minio) which emulates Amazon's S3 service locally.
+This works in conjunction with the `ENVIRONMENT` env variable being set to "dev." When this is set, usage of the `boto3` library will automatically
+set `endpoint_url` to the internal docker IP address of the Minio instance. You will need a few env variables:
+
+```python
+MINIO_ROOT_USER=minio_user
+MINIO_ROOT_PASSWORD=minio_password
+AWS_ACCESS_KEY_ID=minio_user
+AWS_SECRET_ACCESS_KEY=minio_password
+AWS_STORAGE_BUCKET_NAME=ol-ocw-studio-app
+AWS_PREVIEW_BUCKET_NAME=ocw-content-draft
+AWS_PUBLISH_BUCKET_NAME=ocw-content-live
+AWS_ARTIFACTS_BUCKET_NAME=ol-eng-artifacts
+RESOURCE_BASE_URL_DRAFT=https://draft.ocw.mit.edu
+RESOURCE_BASE_URL_LIVE=https://ocw.mit.edu
+```
```

Comment:
These need to be set to functioning URLs for the ocw-www pipeline to complete successfully.

```
OCW_STUDIO_LIVE_URL=https://ocw.mit.edu
OCW_STUDIO_DRAFT_URL=https://draft.ocw.mit.edu
```

or maybe the pipeline variable replacement code could be adjusted to set the value of `static-api-base-url` based on `RESOURCE_BASE_URL_DRAFT` and `RESOURCE_BASE_URL_LIVE` instead when `ENVIRONMENT=dev`?